deepmd/entrypoints/train.py

```diff
@@ -191,6 +220,36 @@ def get_data(jdata: Dict[str, Any], rcut, type_map, modifier):
 
     return data
 
+def get_data_mt(jdata: Dict[str, Any], rcut, type_map, modifier):
+    systems = j_must_have(jdata, "systems")
+    batch_size = j_must_have(jdata, "batch_size")
+    sys_probs = jdata.get("sys_probs", None)
+    auto_prob = jdata.get("auto_prob", "prob_sys_size")
+    total_data = []
+    for sub_sys in systems:
```

Comment:
Shall we move the `for` iteration into the constructor of `DeepmdDataDocker`? Herein, it can has similar constructor interfaces with `DeepmdDataSystem` where `object-oriented programming` is obeyed.

Reply:
okay, I'll get it fixed